src/errors.ts

```diff
@@ -44,6 +43,7 @@ export enum ErrorType {
   NoGuidanceProfileConfigured = 'noGuidanceProfileConfigured',
   AppSignatureRejected = 'appSignatureRejected',
   LicenseLimitExceeded = 'licenseLimitExceeded',
+  SocketTimeout = 'socket_timeout',
```

Comment:
I would suggest to make error type more generic, so we can cover different types of connection/socket/pool timeouts. Sidebar user generally doesn't need to know in detail the technical details of timeout.

RequestTimeout = 'request_timeout'